ctlcmd/viscmd.py

```diff
@@ -159,7 +160,7 @@ def run(self, args):
         reco_x.append(center.x)
         reco_y.append(center.y)
 
-      self.write_standard_line((center.x, center.y), det_id=args.detid)
+      filename = self.write_standard_line((center.x, center.y), det_id=args.detid)
```

Comment:
What does filename need to be saved here?

Reply:
we decided to save the filename in the updated detector data structure, right?

Reply:
Yes, but `self.write_standadrd_line` doesn't return a filename. I think you are thinking of the `self.filename` that you implemented for commands that save files.